src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -73,6 +125,27 @@ public void Max()
             Assert.AreEqual(largerOffset, result);
         }
 
+        [Test]
+        public void Min()
+        {
+            var smallerOffset = Offset.FromHours(3);
+            var largerOffset = Offset.FromHours(5);
+
+            var result = Snippet.For(Offset.Min(smallerOffset, largerOffset));
+
+            Assert.AreEqual(smallerOffset, result);
+        }
+
+        [Test]
+        public void Negate()
+        {
+            var offsetToNegate = Offset.FromHours(2);
+
+            var result = Snippet.For(Offset.Negate(offsetToNegate));
+
+            Assert.AreEqual(result, -offsetToNegate);
```

Comment:
Again, result needs to be second - and I'd use `Assert.AreEqual(Offset.FromHours(-2), result);`